README.md

```diff
@@ -1 +1,15 @@
-# fio-docker
\ No newline at end of file
+# fio-docker
+
+[![docker build](https://img.shields.io/docker/cloud/build/xridge/fio.svg)](https://hub.docker.com/r/xridge/fio)
+[![docker pulls](https://img.shields.io/docker/pulls/xridge/fio.svg)](https://hub.docker.com/r/xridge/fio)
+[![license](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://www.apache.org/licenses/LICENSE-2.0)
+
+fio Docker Image based on Alpine Linux.
+
+## Usage
+```bash
+docker run -v $(pwd)/data:/data -e JOB_FILES=/jobs/rand-rw.fio xridge/fio
```

Comment:
Maybe it's worth pointing out that it also works without mounting /data. Provide two usage examples accordingly?

Reply:
I'm not exactly sure what we are measuring that way (docker layerfs performance maybe?) but added anyway. I also changed the job-file to show more examples.